modules/cmpapi/README.md

```diff
@@ -118,6 +118,19 @@ cmpApi.update(null);
 
 ````
 
+### Options for performance improvements
+The third argument of the CmpApi.update method is an object of options that can be provided
+to improve the performance of skipping some calls to TCString.decode method.
+
+````javascript
+
+// preDecodedTCModel is useful if you already have the relative TCModel of the encodedTCString
+// this will enable you to skip the TCString.decode call
+// decodeWithCache if set to true it will call the TCString.decode using the previously TCModel decoded in cache
+// this skip a lot of works
+cmpApi.update(encodedTCString, false, {preDecodedTCModel: null, decodeWithCache: false});
```

Comment:
I am not sure we can change api without discussion and approval, also these parameters are looks very complicated to manage, is it possible to make this improvement always running or it does not make sense for all cases?

Reply:
The purpose of this additional parameter is not to introduce breaking changes, since in any case the first two parameters that were already used do not change. Honestly, I was careful not to introduce breaking changes to avoid behaviors that those who already use the library might not expect, but we can also evaluate other solutions. Honestly, I haven't introduced mechanisms or classes to avoid adding weight and complexity to the project, but I don't think it's a problem